src/Files.App/Actions/IAction.cs

```diff
@@ -5,17 +5,47 @@ namespace Files.App.Actions
 {
 	public interface IAction
 	{
+		/// <summary>
+		/// A label for display in context menus and toolbars
```

Comment:
It can be displayed in the command palette as well.

Reply:
I rather think that the command palette should display the description.
![image](https://user-images.githubusercontent.com/66369541/227081541-e26ab42e-dbab-4db6-bd53-bd80aa8cfa81.png)

Reply:
👍